plugins/ember-cli-deploy-firebase.svg

```diff
@@ -2,5 +2,6 @@
 layout: plugin
 url: http://github.com/ibroadfo/ember-cli-deploy-firebase
 start_version: "0.6.0"
+end_version: "1.0.0"
```

Comment:
unless you think you won't be compatible with 1.0 (which I think you will given that there are no breaking changes) 

there is no need for the end version